plugins/linuxplugin/ifplugin/interface_config.go

```diff
@@ -681,7 +681,9 @@ func (plugin *LinuxInterfaceConfigurator) removeObsoleteVeth(nsMgmtCtx *nsplugin
 	defer revertNs()
 	if err != nil {
 		// Already removed as namespace no longer exists.
-		plugin.IfIndexes.UnregisterName(vethName)
+		if _, _, found := plugin.IfIndexes.LookupIdx(vethName); found {
+			plugin.IfIndexes.UnregisterName(vethName)
```

Comment:
This should make no difference, since unregistering non-existing name will be ignored anyway and no delete event should be generated.